### DISCUSSION TITLE
Sizing Contribution Feed Elements

### DISCUSSION THREAD

**Discussion by foxyseta** [community/community#27767] [Mobile]
The feed elements describing a user contribution to a repo always have:
- excessive minimum height
- a "Read more button"
even where there's actually nothing to read at all! The following screen was taken on Android:
![Screenshot_20220731-062841.png](https://user-images.githubusercontent.com/12249077/182010121-b49854dd-bdec-4f40-8fd9-4b54f5516324.png)

**Comment by jfullstackdev** [comment]
maybe related to this one 

https://github.com/community/community/discussions/4509

**Reply by foxyseta** [reply]
Yes, perhaps it is.

**Comment by stevepopovich** [comment]
I passed this on to the team for more investigation!
